### PR TITLE
fix: update online debug api protocol validation and error msg

### PR DESCRIPTION
**Pull request by liuxiran**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix

- Related issues
last part of #1077 

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: api/internal/handler/route_online_debug/route_online_debug.go [hunk 1]
```diff
@@ -80,7 +81,8 @@ func DebugRequestForwarding(c droplet.Context) (interface{}, error) {
 	if v, ok := protocolMap[requestProtocol]; ok {
 		return v.RequestForwarding(c)
 	} else {
-		return &data.SpecCodeResponse{StatusCode: http.StatusBadRequest}, fmt.Errorf("protocol unspported %s", paramsInput.RequestProtocol)
+		return &data.SpecCodeResponse{StatusCode: http.StatusBadRequest},
+			fmt.Errorf("protocol unspported %s, need http or https. but given %s", paramsInput.RequestProtocol, paramsInput.RequestProtocol)
```
Comment (juzhiyuan) [inline api/internal/handler/route_online_debug/route_online_debug.go:85]: unsupported

Comment (juzhiyuan) [inline api/internal/handler/route_online_debug/route_online_debug.go:85]:
```suggestion
			fmt.Errorf("Protocol unsupported %s, only http or https is allowed, but given %s", paramsInput.RequestProtocol, paramsInput.RequestProtocol)
```

**Review by juzhiyuan — APPROVED** [review]

File: api/internal/handler/route_online_debug/route_online_debug.go (hunk 1 above)
Comment (liuxiran) [reply api/internal/handler/route_online_debug/route_online_debug.go]: many thanks~

File: api/internal/handler/route_online_debug/route_online_debug.go [hunk 2]
```diff
@@ -49,6 +49,7 @@ var protocolMap map[string]ProtocolSupport
 func init() {
 	protocolMap = make(map[string]ProtocolSupport)
 	protocolMap["http"] = &HTTPProtocolSupport{}
+	protocolMap["https"] = &HTTPProtocolSupport{}
 }
```
Comment (nic-chen) [inline api/internal/handler/route_online_debug/route_online_debug.go:53]: oh, we don't need a init function here, just a map is ok

Comment (nic-chen) [reply api/internal/handler/route_online_debug/route_online_debug.go:53]: you could refer to:
https://github.com/uber-go/guide/blob/master/style.md#avoid-init

Comment (liuxiran) [reply api/internal/handler/route_online_debug/route_online_debug.go:53]: > you could refer to:
> https://github.com/uber-go/guide/blob/master/style.md#avoid-init

thanks, updating~

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**